test/DotNetCore.CAP.MySql.Test/DotNetCore.CAP.MySql.Test.csproj

```diff
@@ -11,6 +11,7 @@
   </ItemGroup>
   
   <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.35" />
```

Comment:
Can this reference be removed? It doesn't seem to be used

Reply:
![image](https://user-images.githubusercontent.com/12774053/84354359-5209d480-abf3-11ea-8baa-d7eb1ee6ebd6.png)